React/wbooks/src/app/screens/BookDetail/index.js

```diff
@@ -38,6 +39,7 @@ class BookDetail extends Component {
     const imageUrl = this.state.book && this.state.book.image_url;
     return (
       <div className={styles.bookDetail}>
+        <Navbar/>
```

Comment:
That isn't OK. U don't have to repeat in each screen the navbar. You should use it in the container of the whole app that does not include the login